### ISSUE TITLE
cirq.StateVectorSimulatorState removed

### ISSUE THREAD

**Issue by maffoo** [quantumlib/qsim#545]
Unable to import qsimcirq with latest versions of cirq, due to `cirq.StateVectorSimulatorState` being removed:
```
  File "/usr/local/google/home/maffoo/.virtualenvs/pyle/lib/python3.8/site-packages/qsimcirq/__init__.py", line 56, in <module>
    from .qsim_simulator import (
  File "/usr/local/google/home/maffoo/.virtualenvs/pyle/lib/python3.8/site-packages/qsimcirq/qsim_simulator.py", line 27, in <module>
    class QSimSimulatorState(cirq.StateVectorSimulatorState):
AttributeError: module 'cirq' has no attribute 'StateVectorSimulatorState'
```

**Comment by 95-martin-orion** [comment]
Yup, `pip install cirq --pre` is going to be pretty unstable for a bit while Cirq 1.0 changes rush in. qsim hasn't released a patch for 1.0 (in no small part because of how many deprecations are going through) so for now you should fall back to stable Cirq (no `--pre`)

**Comment by 95-martin-orion** [comment]
Duplicate of #543
